web/src/js/components/FlowView/Messages.jsx

```diff
@@ -68,17 +68,18 @@ function ResponseLine({ flow, readonly, updateFlow }) {
 
 const Message = connect(
     state => ({
-        flow: state.ui.flow.modifiedFlow || state.flows.byId[state.flows.selected[0]],
+        flow: state.flows.byId[state.flows.selected[0]],
```

Comment:
I think that providing the modified flow is essential here, does Header editing work with this change!? The ContentView/ContentLoader needs to be fixed so that it doesn't rerender.